.travis.yml

```diff
@@ -11,29 +11,29 @@ matrix:
     - php: 7.3
     - php: 7.3
       env: deps=low
+    - php: 7.4
+    - php: 7.4
+      env: deps=low
     - php: nightly
       env: deps=low
     - php: nightly
   allow_failures:
+    - php: 7.4
+      env: deps=low # This will install phpspec v5, which does not support PHP 7.4.
```

Comment:
Maybe we should completely remove the `deps=low` test for PHP 7.4? :thinking:

Reply:
Are you sure it will install PHPSpec v5 since composer will check the PHP version during dependencies resolution ?

Reply:
You are right. This was another issue I addressed here: https://github.com/friends-of-phpspec/phpspec-code-coverage/pull/22#discussion_r365200439